scripts/rollup/build-all-release-channels.js

```diff
@@ -74,7 +74,11 @@ if (process.env.CIRCLE_NODE_TOTAL) {
   buildForChannel('stable', '', '');
   const stableDir = tmp.dirSync().name;
   crossDeviceRenameSync('./build', stableDir);
-  processStable(stableDir);
+  try {
+    processStable(stableDir);
+  } catch (err) {
+    console.log(err);
+  }
```

Comment:
Why do we need this `try/catch` statement here? If `processStable` throws, then we should not continue the script execution

Reply:
I believe I did that for debugging purposes, I have removed that